be/src/gutil/strings/escaping.cc

```diff
@@ -1659,6 +1659,22 @@ static void a2b_hex_t(const char* a, T b, int num) {
     }
 }
 
+template <typename T>
+static bool a2b_hex_t_checked(const char* a, T b, int num) {
+    for (int i = 0; i < num; i++) {
+        char l = ascii_tolower(a[i * 2]);
+        char r = ascii_tolower(a[i * 2 + 1]);
+        if (l < '0' || l > 'f') {
```

Comment:
is this function efficient enough ?

Reply:
like `unhexImpl`. but need to check its safety.